python_modules/dagster/dagster/_core/definitions/time_window_partitions.py

```diff
@@ -679,8 +679,10 @@ def empty_subset(self) -> "PartitionsSubset":
 
     def is_valid_partition_key(self, partition_key: str) -> bool:
         try:
-            time_obj = datetime.strptime(partition_key, self.fmt)
-            return time_obj.timestamp() >= self.start.timestamp()
+            partition_time = pendulum.instance(
+                datetime.strptime(partition_key, self.fmt), tz=self.timezone
+            )
+            return partition_time.timestamp() >= self.start.timestamp()
```

Comment:
now that everything is tz-aware, can we just do partition_time >= self.start?

Reply:
yep! makes sense